Description: Adjusts the classpath of the jar files to point to the Debian files in /usr/share/java
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/jaxws-ri/bundles/jaxws-rt/pom.xml
+++ b/jaxws-ri/bundles/jaxws-rt/pom.xml
@@ -245,7 +245,7 @@
                             <archive>
                                 <manifestEntries>
                                     <probe-provider-class-names>com.sun.xml.ws.transport.http.servlet.JAXWSRIDeploymentProbeProvider</probe-provider-class-names>
-                                    <Class-Path>javax.annotation-api.jar jsr181-api.jar jaxb-api.jar jaxws-api.jar javax.xml.soap-api.jar FastInfoset.jar gmbal-api-only.jar ha-api.jar jaxb-impl.jar management-api.jar mimepull.jar policy.jar saaj-impl.jar stax-ex.jar stax2-api.jar streambuffer.jar woodstox-core-asl.jar</Class-Path>
+                                    <Class-Path>/usr/share/java/geronimo-annotation-1.3-spec.jar /usr/share/java/jws-api.jar /usr/share/java/jaxb-api.jar /usr/share/java/jaxws-api.jar /usr/share/java/javax.xml.soap-api.jar /usr/share/java/FastInfoset.jar /usr/share/java/gmbal.jar ha-api.jar /usr/share/java/jaxb-impl.jar /usr/share/java/glassfish-management-api.jar /usr/share/java/mimepull.jar /usr/share/java/metro-policy.jar /usr/share/java/saaj-impl.jar /usr/share/java/stax-ex.jar /usr/share/java/stax2-api.jar /usr/share/java/streambuffer.jar /usr/share/java/woodstox-core-asl.jar /usr/share/java/xml-resolver.jar</Class-Path>
                                     <Extension-Name>com.sun.xml.ws</Extension-Name>
                                     <Multi-Release>true</Multi-Release>
                                 </manifestEntries>
